job/server/src/main/java/alluxio/master/job/metrics/DistributedCmdMetrics.java

```diff
@@ -229,40 +236,43 @@ private static String getFilePathForNonBatchConfig(JobConfig config) {
   public static void batchIncrementForCompleteStatusWithRetry(
           BatchedJobConfig config, FileSystem fileSystem, RetryPolicy retryPolicy) {
     String jobType = config.getJobType();
-    long count = config.getJobConfigs().size();
+    if (jobType.equals(MigrateConfig.NAME) || jobType.equals(LoadConfig.NAME)
+        || jobType.equals(PersistConfig.NAME)) {
 
-    incrementOperationCount(jobType, count);
-    incrementFileCount(jobType, count);
+      long count = config.getJobConfigs().size();
+      incrementOperationCount(jobType, count);
+      incrementFileCount(jobType, count);
 
-    // filePath is defined for MigrateConfig, LoadConfig and PersistConfig currently.
-    // The "filePath" is the key to the json map inside config to get the actual file path.
-    String pathMapKey = null;
-    if (jobType.equals(MigrateConfig.NAME)) {
-      pathMapKey = "source";
-    } else if (jobType.equals(LoadConfig.NAME) || jobType.equals(PersistConfig.NAME)) {
-      pathMapKey = "filePath";
-    }
+      // filePath is defined for MigrateConfig, LoadConfig and PersistConfig currently.
```

Comment:
same comment here, there is already an equivalent "is this a trackable job type" check here

Reply:
agreed, i forgot to modify the "equivalent checking", will update, thanks